tests/image_test.py

```diff
@@ -1248,6 +1248,37 @@ def test_normalize_channel(fx_asset):
             assert getattr(img[-1, 0], c) == getattr(right_top, c)
             assert getattr(img[-1, -1], c) == getattr(right_bottom, c)
 
+def test_level_default(fx_asset):
+    im_qr = pow(2, QUANTUM_DEPTH)
```

Comment:
Folks are going to be mad if they have to calculate quantum values, also this is the systems constant (see Image.quantum_depth property.)